clusterware-customize/libexec/customize/actions/help

```diff
@@ -152,6 +153,42 @@ help_for_trigger() {
 EOF
 }
 
+help_for_push() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces customize push
+
+  DESCRIPTION:
+
+    Push a customization profile to an upstream repository.
+EOF
+}
+
+help_for_slave() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces customize slave [OPTIONS]
+
+  DESCRIPTION:
+
+    Modifies the list of profiles to be ran when a compute node boots.
+    Has the option to add, remove or list the profiles.
+
+  OPTIONS:
```

Comment:
Either that or we do:

```
SYNOPSIS:

  alces customize slave add <profile>
  alces customize slave remove <profile>
  alces customize slave list

DESCRIPTION:

  List the currently enabled customization profiles to be executed by slave nodes on boot, or add or remove a customization <profile> from the centrally managed list on the master node.
```

That might be easiest as the tool isn't hugely complex -- yet! :)